packages/components/src/internal/components/entities/EntityMoveConfirmationModal.spec.tsx

```diff
@@ -144,9 +144,11 @@ describe('EntityMoveConfirmationModal', () => {
         await waitForLifecycle(wrapper);
         expect(wrapper.find(Modal)).toHaveLength(1);
         expect(wrapper.find(SelectInput)).toHaveLength(1);
-        expect(wrapper.find(SelectInput).prop('options').length).toBe(1);
+        expect(wrapper.find(SelectInput).prop('options').length).toBe(2);
```

Comment:
add test case for the new `excludeCurrentAsTarget` prop?

Reply:
This is getting rewritten in RTL.